src/frontend/main.go

```diff
@@ -84,7 +84,7 @@ type frontendServer struct {
 
 func main() {
 	ctx := context.Background()
-	log := logrus.New()
+	log := logrus.initializeLogger()
 	log.Level = logrus.DebugLevel
```

Comment:
Issue: We should remove these lines:
```
log.Level = logrus.DebugLevel
	log.Formatter = &logrus.JSONFormatter{
		FieldMap: logrus.FieldMap{
			logrus.FieldKeyTime:  "timestamp",
			logrus.FieldKeyLevel: "severity",
			logrus.FieldKeyMsg:   "message",
		},
		TimestampFormat: time.RFC3339Nano,
	}
	log.Out = os.Stdout
```
since they're being taken care of by `initializeLogger()`.